### ISSUE TITLE
[BUG] Update failing to install

### ISSUE THREAD

**Issue by muness** [mrjackphil/obsidian-text-expand#37]
I was trying to test the latest release to get the fix for #35 but I am getting an error when updating through the UI:

```
https://github.com/mrjackphil/obsidian-text-expand/releases/download/0.8.1/manifest.json 
Failed to load resource: the server responded with a status of 404 ()
```

![image](https://user-images.githubusercontent.com/931/114185520-d14f5e00-9913-11eb-9156-ce06091b176b.png)

I checked https://github.com/mrjackphil/obsidian-text-expand/releases/download/0.8.1/manifest.json and it indeed returns a 404.

**Comment by muness** [comment]
I see that you fixed the release in 0.8.2, but something must be pointing to 0.8.1 as the latest. I don't know enough about how Obsidian's plugin framework works to localize the problem further.

**Comment by mrjackphil** [comment]
@muness Sorry, my mistake. Should work now.
